settings.py

```diff
@@ -2061,3 +2061,5 @@ def _pkce_required(client_id):
 GOOGLE_SHEETS_API_VERSION = "v4"
 
 DAYS_KEEP_GSHEET_STATUS = 14
+COMMCARE_ANALYTICS_HOST = "https://commcare-analytics.dimagi.com" \
```

Comment:
Isn't the usual way to do this to set it up in commcare-cloud? Plus then localhosters won't be redirecting to http://commcare-analytics-demo.dimagi.com/

Reply:
Oooh, good point.

Reply:
yep, exactly